### ISSUE TITLE
custom button 

### ISSUE THREAD

**Issue by armanenb** [YanYuanFE/react-native-signature-canvas#110]
can i use it without footer part , and add my cuson ok/clear buttons

**Comment by YanYuanFE** [comment]
Yes, you can use webStyle prop via display: none to hidden button element, and add React Native Button Component to control read signature or clear signature.https://github.com/YanYuanFE/react-native-signature-canvas#methods
https://github.com/YanYuanFE/react-native-signature-canvas#basic-parameters;
```
const webStyle = `.m-signature-pad--footer
    .save {
        display: none;
    }
    .clear {
        display: none;
    }
`;
...
  <Signature
    webStyle={webStyle}
    onOK={handleOK}
    onEmpty={handleEmpty}
    onEnd={handleEnd}
  />
```

**Comment by YanYuanFE** [comment]
https://github.com/YanYuanFE/react-native-signature-canvas#custom-button-for-confirm-and-clear
